src/CentralBank.java

```diff
@@ -0,0 +1,128 @@
+import java.math.BigDecimal;
+import java.util.*;
+
+public class CentralBank
+{
+    private List<Account> accounts;
```

Comment:
```suggestion
    private final List<Account> accounts = new ArrayList<>();
```
필드에 선언하는 것이 좋을 것 같습니다.
final키워드를 선언함으로써 변경되지 않을 것을 명시해줘야합니다